queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered=true and escape_attempts<3;
```

Comment:
- [ ] For the second, third and fourth queries you should list the name of all animals born between 2016 and 2019, kindly make sure to change` SELECT * FROM animals...` to `SELECT name FROM animals...`

Reply:
@NurkaAmre, You need to select only the `name` from the animals table, not all the columns `*`